portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,225 @@
-#content {
+p {
+  font-size: 40px
+}
+#logo{
+  text-align: center;
+  background-image: url("https://media.giphy.com/media/4rwQehrSJTS6s/giphy.gif");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+#lll {
+  font-size: 170px;
+  text-transform: uppercase;
+  text-shadow: 3px 0 #30075f, 0 3px #30075f, 3px 0 #30075f, 0 3px #30075f;
+  font-family: 'Cinzel Decorative', cursive;
+  
+}
+#c1{
+  color: rgb(115, 0, 255);
+}
+#c12{
+    color: rgb(115, 0, 255);
+    font-size: 120px;
+}
+#c2{
+  color: rgb(142, 51, 253);
+}
+#c23 {
+  color: rgb(174, 76, 255);
+  font-size: 120px;
+}
+#c3{
+  color: rgb(180, 104, 242);
+}
+#c4 {
+  font-size: 100px;
+   color: rgb(203, 140, 255);
+}
+#c45 {
+  font-size: 80px;
+  color: rgb(209, 156, 252);
+}
+#c5{
+    color: rgb(221, 181, 255);
+}
+#c6{
+  color: rgb(226, 201, 255);
+  font-size: 120px;
+}
+h1 {
+  display: inline-block;
+}
+#subhead {
+  text-align: center;
+  font-size: 30px;
+  font-family: 'Cinzel Decorative', cursive;
+  color: white;
+  background-color:  black;
+}
+body {
+  margin: 0;
+    padding: 0;
+  background-repeat: no-repeat;
+   background-size:     cover; 
+   background-color: rgb(226, 201, 255);
+}
+/* ul {
+  margin: 0px 0px 0px 0px;
+  padding: 0px 0px 0px 0px;
+  list-style: none;
+  display: inline;
+  font-size: 50px;
+  margin-top: 0;
+  text-align: left;
+  
+  float: left;
+} */
+
```

Comment:
Just some general comments about this file, but otherwise it looks great!

- You've got some whitespace misalignment.  I don't know of any git or cloud shell tools to fix this automatically, but do what you can manually.
- It seems like you're leaning a _little_ too heavily on id selectors.  There may be a way to simplify this file a bit by moving some of the repeated bits to class selectors.
- The sorting in this file makes it a little difficult to parse.  Generally I would group element, id, and class selectors together to make conflicts more visible.

Reply:
Resolving because I used more class and div selectors and adjusted whitespace